dll.py

```diff
@@ -139,11 +235,17 @@ def find(self, value: T) -> "Node":
         return None
 
     def __len__(self) -> int:
-        """Number of nodes in the doubly linked list"""
+        """
+        Returns the length of the list
+
+        :return: number of nodes in the list
+        """
         return self._size
 
     def __str__(self):
-        """Nice representation, needs to loop over all elements!"""
+        """
+        Returns a string representation of the list
+        """
         txt = "["
         node = self._head
         while node is not None:
```

Comment:
Documentation for __iter__ en __reversed:
"Returns an iterator for the values of the nodes in the list"
"Returns a reversed iterator for the values of the nodes in the list"

Iets beters kan ik er niet van maken.